providers/dns/nearlyfreespeech/internal/client_test.go

```diff
@@ -83,7 +88,12 @@ func TestClient_AddRecord(t *testing.T) {
 		"ttl":  "30",
 	}
 
-	mux.Handle("/dns/example.com/addRR", testHandler(params))
+	// The reason we're testing that we're getting this exact authHeader is
+	// so that we can detect if some change that is not expected to modify
+	// neither the request nor the authHeader does so. If you're changing
+	// the request, please ensure your change actually works against the
+	// real nfsn API before modifying this golden value.
```

Comment:
```suggestion
	// The reason we're testing that we're getting this exact authHeader is
	// so that we can detect if some change that is not expected to modify
	// neither the request nor the authHeader does so. If you're changing
	// the request, please ensure your change actually works against the
	// real nfsn API before modifying this golden value.
```